.gitignore

```diff
@@ -19,7 +20,9 @@ plugins/*
 !plugins/wp_import/**
 **/node_modules
 **/node_modules/**
-config.js
 config.json
+coverage/
```

Comment:
I have removed the duplicate entry

From: Brian <notifications@github.com<mailto:notifications@github.com>>
Reply-To: pencilblue/pencilblue <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, July 9, 2015 at 5:13 PM
To: pencilblue/pencilblue <pencilblue@noreply.github.com<mailto:pencilblue@noreply.github.com>>
Cc: Ben Tidwell <ben.tidwell@careerbuilder.com<mailto:ben.tidwell@careerbuilder.com>>
Subject: Re: [pencilblue] Multi Tenancy (#680)


In .gitignore<https://github.com/pencilblue/pencilblue/pull/680#discussion_r34304272>:

>  config.json
> +coverage/


Why was coverage added twice?

-
Reply to this email directly or view it on GitHub<https://github.com/pencilblue/pencilblue/pull/680/files#r34304272>.